src/cascadia/TerminalSettingsEditor/GlobalAppearance.xaml

```diff
@@ -84,6 +84,16 @@
             <local:SettingContainer x:Uid="Globals_DisableAnimationsReversed">
                 <ToggleSwitch IsOn="{x:Bind local:Converters.InvertBoolean(State.Globals.DisableAnimations), BindBack=State.Globals.SetInvertedDisableAnimationsValue, Mode=TwoWay}" />
             </local:SettingContainer>
+
+            <!--  Always Show Tray Icon  -->
+            <local:SettingContainer x:Uid="Globals_AlwaysShowTrayIcon">
+                <ToggleSwitch IsOn="{x:Bind State.Globals.AlwaysShowTrayIcon, Mode=TwoWay}" />
+            </local:SettingContainer>
+
+            <!--  Minimize To Tray  -->
+            <local:SettingContainer x:Uid="Globals_MinimizeToTray">
+                <ToggleSwitch IsOn="{x:Bind State.Globals.MinimizeToTray, Mode=TwoWay}" />
+            </local:SettingContainer>

```

Comment:
FWIW we should want to make these disappear using the same feature key as the tray icon is hidden behind ... otherwise the settings will ship to Stable but not _do_ anything!